tests/integration/modules/test_idem.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""
+Integration tests for the idem execution module
+"""
+
+# Import futures
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
@s0undt3ch do we still need these? That might be an obsolete lint rule we need to remove?

Reply:
I have a draft PR that also takes care of those, but yes, both the coding line and the future imports.

Reply:
Should this merge after that?

Reply:
https://github.com/saltstack/salt/pull/57936

Reply:
Not required. I'm not removing any code, I added a pre-commit rule that removes them for each edited file, to avoid massive changes in a single PR.